config/postBuild.js

```diff
@@ -0,0 +1,4 @@
+const fs = require('fs');
+
+fs.copyFileSync('README.md', 'dist/README.md');
+console.log('README.md was copied to dist');
```

Comment:
missing new line

Reply:
Where? End of file?
Shouldn't prettier take care of that?

Reply:
Good point! Of course, it should!
I'm going to open the issue.

Reply:
We don't really use plain JavaScript in this repository so that's why prettier didn't do it. Also we don't run it automatically which actually is an issue